dev/com.ibm.websphere.appserver.microProfile-1.3/com.ibm.websphere.appserver.microProfile-1.3.feature

```diff
@@ -0,0 +1,24 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.microProfile-1.3
+visibility=public
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-ShortName: microProfile-1.3
+Subsystem-Version: 7.0.0
+Subsystem-Name: Micro Profile 1.3
+-features=com.ibm.websphere.appserver.servlet-3.1; ibm.tolerates:=4.0, \
+ com.ibm.websphere.appserver.javax.annotation-1.2; ibm.tolerates:=1.3, \
+ com.ibm.websphere.appserver.cdi-1.2; ibm.tolerates:=2.0, \
+ com.ibm.websphere.appserver.jaxrs-2.0; ibm.tolerates:=2.1, \
+ com.ibm.websphere.appserver.jsonp-1.0; ibm.tolerates:=1.1, \
+ com.ibm.websphere.appserver.mpConfig-1.2, \
+ com.ibm.websphere.appserver.mpFaultTolerance-1.0, \
+ com.ibm.websphere.appserver.mpHealth-1.0, \
+ com.ibm.websphere.appserver.mpJwt-1.0, \
+ com.ibm.websphere.appserver.mpMetrics-1.0, \
+ com.ibm.websphere.appserver.mpOpenAPI-1.0, \
+ com.ibm.websphere.appserver.mpOpentracing-1.0, \
```

Comment:
I also just verified with Steve Fontes that this will be changing to mpOpenTracing-1.0.  Go ahead with what you have.  I'll have them update this file with their PR...

Reply:
I was aware this was coming, but didn't want to wait. Although the build seems a little sick so they might beat this. I'll watch out.